### PR TITLE
Restore regular YARP branches

### PR DESCRIPTION
**Pull request by traversaro**
As https://github.com/robotology/yarp/pull/2356 has been merged.

### CODE REVIEW

File: cmake/ProjectsTagsStable.cmake [hunk 1]
```diff
@@ -9,6 +9,6 @@ set_tag(osqp_TAG v0.6.0)
 
 # Robotology projects
 set_tag(YCM_TAG ycm-0.11)
-# set_tag(YARP_TAG yarp-3.4)
+set_tag(YARP_TAG yarp-3.4)
 set_tag(YARP_TAG 0328577b5f102da2875517898d8a6439ac9374d2)
```
Comment (S-Dafarra) [inline cmake/ProjectsTagsStable.cmake:13]: Probably this has to be removed

**Review by S-Dafarra — CHANGES_REQUESTED** [review]

File: cmake/ProjectsTagsStable.cmake (hunk 1 above)
Comment (traversaro) [reply cmake/ProjectsTagsStable.cmake]: Thanks, done!

**Review by S-Dafarra — APPROVED** [review]

**Comment by traversaro** [comment]
Apparently the change in https://github.com/robotology/yarp/pull/2356 still needs to be propagated to `master`, so we need to wait for that to merge this PR, because the `Stable` branches give an error as the default branch of YARP is `master`, and the error emerges before switching to `yarp-3.4`.

**Comment by traversaro** [comment]
The automatic merging of `yarp-3.4` in `master` was broken by exactly the same problem described in https://github.com/robotology/yarp/pull/2356 . Now the merge has been done manually, so the PR is ready to be merged.

**Pull request merged**